src/app/pages/home/rolodex/rolodex.page.ts

```diff
@@ -128,19 +133,67 @@ export class RolodexPage implements OnInit {
   }
 
   constructor(private authService: AuthService, private userService: UserService, private cardService: CardService,
-    private route: ActivatedRoute, private router: Router
-  ) {
+              private route: ActivatedRoute, private router: Router, platform: Platform, private screenOrientation: ScreenOrientation,
+              private gyroscope: Gyroscope) {
+
+
     console.log("Rolodex Page (constructor)");
 
     this.staticCards = this.cardService.listStaticCardsByHolder(this.authService.getCurrentUserId());
     this.cards = this.cardService.listCardsByHolder(this.authService.getCurrentUserId());
+    this.users = this.userService.listUsers();
+
+    platform.ready().then(() => {
+    this.getScreenOrientation();
+    }).catch(err => {
+         console.log('Error while loading platform', err);
+    });
+    this.getGyroscopeData();
   }
 
   ngOnInit() {
     console.log("Rolodex Page (init)");
-    this.users = this.userService.listUsers();
   }
 
+  getScreenOrientation(){
```

Comment:
Can we borrow these methods from the parent `home.page.ts`?

Reply:
I don't see why not. Via an external service?

Reply:
That would actually be better. Set up a screen service and gyroscope service and share their methods to any page that needs it with an import like any other service.

Reply:
That should def a part of the next refactoring/code review.